src/earn/EarnEntrypoint.tsx

```diff
@@ -55,29 +97,33 @@ const styles = StyleSheet.create({
     marginBottom: Spacing.Thick24,
   },
   touchable: {
-    padding: Spacing.Regular16,
     borderColor: Colors.gray2,
     borderWidth: 1,
     borderRadius: 8,
   },
   title: {
     ...typeScale.labelSemiBoldMedium,
     color: Colors.black,
+    marginBottom: Spacing.Smallest8,
   },
   row: {
     flexDirection: 'row',
-    marginTop: Spacing.Regular16,
     gap: Spacing.Smallest8,
   },
-  subtitleContainer: {
+  textContainer: {
     flex: 1,
-  },
-  subtitle: {
-    ...typeScale.labelSemiBoldSmall,
-    color: Colors.black,
+    padding: Spacing.Regular16,
   },
   description: {
     ...typeScale.bodySmall,
-    color: Colors.gray4,
+    color: Colors.gray3,
+  },
+  totalSupplied: {
+    ...typeScale.labelSemiBoldMedium,
+    color: '#6168DC',
```

Comment:
have we confirmed with design if this is a one off color or this is going to be used more commonly? Wonder if still worth adding to Colors, so that we can potentially change this easily w/ mobile stack

Reply:
checked with Kayla and she said she just wants it to be black instead